### ISSUE TITLE
Add README and LICENSE

### ISSUE THREAD

**Issue by okurz** [os-autoinst/os-autoinst-distri-opensuse#845]
os-autoinst-distri-opensuse should include some minor doc and information.

#### My proposal:
 * *README*: Could be very similar to or based on https://github.com/kobliha/openqa-yast/blob/master/README.md including a mini contribution note like last line from https://github.com/os-autoinst/os-autoinst/blob/master/README.asciidoc
 * *LICENSE* (or COPYING): A bit tough, ask all developers what LICENSE applies or should be applied. If everyone agrees use GPLv2 like openQA. Could also be permissive like MIT/BSD "2 clause" as it is only test cases anyway :-)
 * We might not want to apply the same for some tests, e.g. SLE in the corresponding branches. Then I would propose to actually separate them into individual github projects

##### proposal for contribution note, license
```
## How to contribute

Fork the repository and make some changes.
Once you're done with your changes send a pull request. Thanks!
If you have questions, visit us on irc.freenode.net in #opensuse-factory or ask on our mailing list opensuse-factory@opensuse.org


## License

Distributed under the XXX license.
```

What are your opinions?

**Comment by coolo** [comment]
there are no tests in the SLE branch and all tests are copyright SUSE, so only SUSE decides what it is.

**Comment by okurz** [comment]
ok, then a README should state that copyright is with SUSE and that contributions by non-SUSE contributors also take on the same license, right?

**Comment by coolo** [comment]
Whatever the license is, contributors will have to either accept it for their contributors - or fork. But right now it doesn't have a license, so it's purely owned by SUSE. We should give it an open source license though as we want other people to take and extend our tests.

**Comment by okurz** [comment]
proposal for README https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/850

**Comment by okurz** [comment]
Fixed with #918 #850
